test/e2e/apimachinery/custom_resource_definition.go

```diff
@@ -339,7 +339,9 @@ var _ = SIGDescribe("CustomResourceDefinition resources [Privileged:ClusterAdmin
 		}}, metav1.CreateOptions{})
 		framework.ExpectNoError(err, "creating CR")
 		v, found, err := unstructured.NestedFieldNoCopy(u2.Object, "a")
-		framework.ExpectEqual(found, true, "\"a\" is defaulted")
+		if !found{
+			framework.Failf("value of object in %+v with field a is nil or it does not exists or its type is not expected type map[string]interface", u2.Object)
```

Comment:
@pohly not very sure about this output, could you advise a better format of output here.

Reply:
```suggestion
			framework.Failf("field `a` should have been defaulted in %+v", u2.Object)
```

A developer reading this should know what "defaulted" means. It also was what the original message said.